exampleCourse/questions/element/dropdown/question.html

```diff
@@ -0,0 +1,20 @@
+	<p> Select the correct word in the following quotes:</p>
+	The 
+	<pl-dropdown sort="ascend" answers-name="aristotle" blank="true">
```

Comment:
Let's just directly embed the options from `server.py` here.

I think there is also a tabbed space here too? Can that be removed?